src/entities/cron.js

```diff
@@ -177,10 +182,8 @@ class CronScheduler implements Scheduler {
           this.logger.debug('Finished loop for stable epochs. Pushing any cached blocks to storage.')
           await this.pushCachedBlocksToStorage()
         } else {
-          // Packed epoch is not available yet
-          this.logger.info(`cardano-http-brdige has not yet packed stable epoch: ${epoch} (lastRemStableEpoch=${lastRemStableEpoch})`)
+          this.logger.info('Jormungandr do not support epoch processing.')
```

Comment:
It shouldn't even be here with jormun since this entire epoch thing is only enabled for byron. See the`if (this.networkProtocol === NETWORK_PROTOCOL.BYRON && epoch < remEpoch) {`